src/event_pusher/mod_event_pusher_hook_translator.erl

```diff
@@ -57,10 +58,17 @@ filter_local_packet({From, To = #jid{lserver = Host}, Acc0, Packet}) ->
     Acc = case chat_type(Acc0) of
               false -> Acc0;
               Type ->
-                  Event = #chat_event{type = Type, direction = out,
-                                      from = From, to = To, packet = Packet},
-                  NewAcc = mod_event_pusher:push_event(Acc0, Host, Event),
-                  merge_acc(Acc0, NewAcc)
+                  case mongoose_lib:is_to_room(To) of
```

Comment:
Actually, `Type` is the message type, that is, it can be the atom groupchat, so you might as well filter there too, you might want to call `is_to_room` only if the message is actually to a groupchat.

Reply:
Yeah, true